src/standalone.js

```diff
@@ -38,17 +38,35 @@ export function loadBuiltin(builtinTable, name) {
   return name;
 }
 
+function trimBabelPluginPrefix(obj) {
```

Comment:
Interesting function! I'm curious as to why we need to do this

Reply:
Take @babel/plugin-proposal-object-rest-spread for example:

When we call `babel.registerPlugin("@babel/plugin-proposal-object-rest-spread", someFunc)` `babel/standalone` will sometimes (can't make this happen consistently) trim `@babel/plugin-` and register the plugin as `proposal-object-rest-spread`. So when the user uses `@babel/plugin-proposal-object-rest-spread` in the config, babel reacts like [this](https://i.redd.it/zrsi2k5469t41.png) and the whole system breaks, no bueno! 

So what `trimBabelPluginPrefix` does is the trimming for us! So we try twice to find the registered plugin, once with the original name and again with the trimmedPrefix. If we can't find it in those two times we bail out and throw an error.

Reply:
Thanks for explanation! 🤯